feature/networkinstance/networkinstance_test.go

```diff
@@ -26,6 +26,11 @@ import (
 	"github.com/openconfig/ygot/ygot"
 )
 
+var protocolKey = oc.NetworkInstance_Protocol_Key{
```

Comment:
you can declare this inside the TestAugment() and not required to be global for this file.

Reply:
Done!